package com.min.edu;

public class ExceptionMain {

	public static void main(String[] args) throws UserDefindException {
//		Exception_Step01 step01 = new Exception_Step01();
//		System.out.println(step01.calculation());
//		Exception_Step02 step02 = new Exception_Step02();
//		System.out.println(step02.calculation());
//		Exception_Step03 step03 = new Exception_Step03();
//		System.out.println(step03.calculation());
//		Exception_Step05 step05 = new Exception_Step05();
//		System.out.println(step05.calculation());
//		Exception_Step06 step06 = new Exception_Step06();
//		System.out.println(step06.calculation());
//		Exception_Step07 step07 = new Exception_Step07();
//		try {
//			System.out.println(step07.calculation(10));
//		} catch (UserDefindException e) {
//			e.printStackTrace();
//		}
//		System.out.println("예외 처리 처리");
		
//		Exception_Step08 step08 = new Exception_Step08();
//		try {
//			step08.makeException();
//		} catch (Throwable e) {
//			e.printStackTrace();
//		}
		
		Exception_NotUsed nn = new Exception_NotUsed();
		System.out.println(nn.calculation());
		
	}

}








